scipy/signal/filter_design.py

```diff
@@ -34,18 +34,22 @@ class BadCoefficients(UserWarning):
 abs = absolute
 
 
-def findfreqs(num, den, N):
+def findfreqs(num, den, N, input='ba'):
```

Comment:
I agree that gh-3259 is out of scope here, and adding a keyword to select ``'zpk'`` isn't going to get in the way of a new API, whatever we choose in gh-3259.